draft-scenarios/coops/the_ambushed_accord.tex

```diff
@@ -0,0 +1,119 @@
+
+\begin{multicols}{2}
+
+\textbf{Author:} Wololoeren
+
+\textit{The war had dragged on far too long. Armies were shattered, lands burned, and hope had dwindled to barely a whisper. At last, the rival lords agreed to set aside their grievances and forge an Accord — a fragile peace born of desperation.\\
+Messengers rode beneath white banners, oaths were sworn, and for the first time in years, the clash of steel gave way to uneasy silence. Yet peace has its own enemies.\\
+As the allied hosts gathered in the valley of Laren's Cross, dawn broke with the sound of horns — not of welcome, but of war.\\
+Out of the mist, enemy armies struck with deadly precision. Someone had betrayed the Accord.\\Now, surrounded and outnumbered, the fate of your alliance lies in your hands. Stand together — or fall alone.}  % no-check-caps
+\subsection*{\MakeUppercase{Scenario Length}}
+14 Rounds
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2-6
+
+\textbf{Starting Resources:} 2 \svg{gold}, 3 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+
+\begin{itemize}
+  \item 2 × Search (2) the \svg{bronze} Neutral Deck and 1 x \svg{morale_positive}
+\end{itemize}
+
+\textbf{Town Buildings:} City Hall
+
+\textbf{Map Tile Pool:} Every player takes 2 random Far (II-III) Map Tiles and 2 random Near (IV-V) Map Tiles.
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles ($P$ stands for the number of players) and arrange them as shown in the Scenario map layout:
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tiles
+  \item 1 × Center (VI–VII) Map Tile
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Every player must defeat the Level VII Neutral Combat.
+
+\subsection*{\MakeUppercase{Timed Events}}
+\textbf{\nth{2} Round:}
+\begin{itemize}
+  \item Build either Camp \svg{building_special_tent} or Workshop \svg{building_special_gears} for free. %(\svg{bulding_special_gears})
+\end{itemize}
+End of \textbf{\nth{4} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 5 × \svg{bronze}.
+\end{itemize}
+End of \textbf{\nth{7} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 4 × \svg{silver}
+\end{itemize}
+End of \textbf{\nth{9} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 3 × \svg{golden}.
+\end{itemize}
+End of \textbf{\nth{11} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 2 × \svg{azure}.
+\end{itemize}
+End of \textbf{\nth{13} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 3 × \svg{azure}.
+\end{itemize}
+\clearpage
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\hommtablemulticol[]{22}{
+  \centering
+  \medskip
+  \textbf{Difficutly Settings}\\
```

Comment:
Normally, tables with difficulty settings are at the end of the scenario, so as not to disrupt the flow of text. How about making all the rulings separate sections? Firstly, timed event fights, as they are the most important. Then, additional rules, and finally, difficulty table and map layout. What do you think?

<img width="1241" height="1754" alt="the_ambushed_accord-02" src="https://github.com/user-attachments/assets/3e74b3ea-d29c-4897-af5e-0eb818b6d497" />